public/locales/de/common.json

```diff
@@ -0,0 +1,22 @@
+{
+    "logo": {
+        "alt": "Instant Visio Logo."
+    },
+    "homeBaseline": "Rufen Sie einen Verwandten an, mit einem Klick, <1>kostenlos.</1>",
+    "headerBaseline": "Rufen Sie einen Verwandten an, mit einem Klick, kostenlos.",
+    "footer": {
+        "contact-us": "Uns kontaktieren",
+        "legal-mentions": "Rechtliche Hinweise",
+        "personal-data": "Personenbezogene Daten",
+        "blog": "Blog",
+        "credits": "Nachweise",
+        "contact": "Kontaktaufnahme"
+    },
+    "url": {
+        "legal-mentions": "mentions-legales",
+        "personal-data": "donnees-personnelles",
+        "blog": "blog",
+        "blog-article": "blog/:article",
```

Comment:
As changes were since made to the French version of this file (available here: https://github.com/Instant-Visio/InstantVisio-WebApp/blob/81ab76abe8653c9f972035ff87b4bccf7f3d1e6b/public/locales/de/common.json), could you, in the url object:

- remove the `"blog-article"` key
- add `"video-call": "visio"`

Reply:
ok